src/game/protos/player.c

```diff
@@ -146,16 +146,16 @@ void player_describe_frame(const sd_script_frame *frame) {
 
 void player_describe_object(object *obj) {
     log_debug("Object:");
-    log_debug("  - Start: %d, %d", obj->start.x, obj->start.y);
-    log_debug("  - Position: %d, %d", obj->pos.x, obj->pos.y);
-    log_debug("  - Velocity: %d, %d", obj->vel.x, obj->vel.y);
+    log_debug("  - Start:    %f, %f", fixedpt_tofloat(obj->start.fx), fixedpt_tofloat(obj->start.fy));
+    log_debug("  - Position: %f, %f", fixedpt_tofloat(obj->pos.fx), fixedpt_tofloat(obj->pos.fy));
+    log_debug("  - Velocity: %f, %f", fixedpt_tofloat(obj->vel.fx), fixedpt_tofloat(obj->vel.fy));
```

Comment:
This is the less verbose way of printing a fixed point number-- by converting it to a float, we can use float format strings.